qiita_pet/support_files/doc/source/checklist-for-ebi-ena-submission.rst

```diff
@@ -0,0 +1,79 @@
+Checklist to send data to EBI-ENA
+=================================
+
+For each preparation that needs to be uploaded to EBI-ENA we will check:
+
+  1. Data processing
+
+    a. Make sure there is raw data linked - no BIOMs allowed
```

Comment:
`no BIOMs allowed` gives the impression that users are not allowed to generate BIOM data.

Reply:
What about: "Only datasets where raw sequences are available and linked to the preparation can be submitted. Studies where the starting point is a BIOM table cannot be submitted, since EBI is a sequence archive." or something along those lines.